Author: Guillem Jover <guillem@debian.org>
Description: Use correct pathname for pci.ids on Debian systems.

---
 cmd/systool.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/cmd/systool.c
+++ b/cmd/systool.c
@@ -655,7 +655,7 @@ int main(int argc, char *argv[])
 	char *show_root = NULL;
 	int retval = 0;
 	int opt;
-        char *pci_id_file = "/usr/local/share/pci.ids";
+	char *pci_id_file = "/usr/share/misc/pci.ids";
 	
 	while((opt = getopt(argc, argv, cmd_options)) != EOF) {
 		switch(opt) {
